frontend/build.ts

```diff
@@ -39,14 +39,17 @@ async function runBuild(dev: boolean) {
     format: "iife",
     bundle: true,
     loader: {
-      ".eot": "file",
-      ".ttf": "file",
-      ".svg": "file",
-      ".woff": "file",
+      ".eot": "empty",
```

Comment:
Very clever!

It does result in CSS with a bunch of empty `url()`s like:
```css
/* node_modules/font-awesome/css/font-awesome.css */
@font-face {
  font-family: FontAwesome;
  src: url();
  src:
    url() format("embedded-opentype"),
    url(./fontawesome-webfont-5GKVPAEF.woff2?v=4.7.0) format("woff2"),
    url() format("woff"),
    url() format("truetype"),
    url() format("svg");
  font-weight: 400;
  font-style: normal;
}
```

That looks a bit weird, but doesn't seem to cause trouble...